Rsk.Samples.IdentityServer4.AdminUiIntegration/Startup.cs

```diff
@@ -39,6 +40,13 @@ public Startup(IWebHostEnvironment env)
 
         public void ConfigureServices(IServiceCollection services)
         {
+            //add memory cache for custom identity server event sink
+            //limit cache to preserving the last 10 error or failure identity server events
+            services.AddMemoryCache(options =>
+            {
+                options.SizeLimit = 10;
```

Comment:
**question:** what's the different between the size set here and the size used in the event store

Reply:
The SizeLimit set in the StartUp is an arbitrary value that is then contextualised by setting the size of something when adding it to the cache. So because all I'm using the cache for is to store the error/failure JSON and whenever I store one of those objects in the cache I give it a size of 1. I'm saying that the cache has a limit of 10 stored error/failure JSON objects before it's trimmed.